crates/holochain/src/core/workflow/call_zome_workflow.rs

```diff
@@ -32,7 +32,7 @@ pub type ZomeCallResult = RibosomeResult<ZomeCallResponse>;
 
 pub struct CallZomeWorkflowArgs<Ribosome>
 where
-    Ribosome: RibosomeT + Send,
+    Ribosome: RibosomeT + Send + Sync,
```

Comment:
It seems as though we're pretty much requiring any `RibosomeT` to be `Send + Sync` in order for it to work with holochain.

I'd recommend promoting those bounds to the type itself, e.g.:

```rust
pub trait RibosomeT: 'static + Send + Sync {
```

Because the errors become much more clear when implementing / changing the ribosome if you do something to violate one of those constraints.